docs: intro sentence for date-dropdown (#1605)
siemens/ix: packages/documentation/docs/controls/buttons/link-button.md

# Link button

<Tags />
<!-- introduction start -->

> Link buttons lead users to another location within the application or outside of it and  contain a chevron and a text label.

Reviewer: ```suggestion
Link buttons take users to another location either within or outside the application and contain a chevron and a text label.
```
